src/webhooks/mod.rs

```diff
@@ -196,13 +238,25 @@ pub fn send(conn: &impl postgres::GenericConnection, task: &worker::TaskType) ->
             }
         };
 
-        match client.post(hook.url.as_str())
+        let mut mac = match HmacSha256::new_varkey(hook.secret.unwrap().as_bytes()) {
```

Comment:
Is it worth handling the case where hook.secret is None here? I see that in a secret value is always created in the create function, but what about existing webhooks? Just want to ensure we don't panic in that case.

Reply:
The `secret` field in the webhooks table is guaranteed to be a non null value https://github.com/mapbox/Hecate/pull/189/files#diff-b22b92c12d5643a0424ec26c1b8e6262R13, which means I'll need to update all existing webhooks anyway. The reason it's an `Option` is so we can handle generating it for the user on `create` if they don't provide, but all functions that retrieve the webhook from the database are guaranteed to have a value for the secret. There may be a different way to handle this than `unwrap`, but it seems appropriate to panic if for some reason that constraint is violated.

Reply:
Nope, that sounds good to me!